Use GKE version 1.25. Remove PSP and istio from GKE cluster creation.

soumyapani opened the pull request.

soumyapani commented on README.md:
```diff
@@ -46,20 +45,20 @@ The optional parameters are:
     -  The `<NFS fileshare type>` cannot be empty. The supported values are `BASIC_HDD`,`BASIC_SSD`,`HIGH_SCALE_SSD` and `ENTERPRISE`.
     -  The `<NFS fileshare size in GB>` can be empty and the default value is 2560 GB (2.5 TB).
-1. ***SHOW_PROXY_URL***. This controls if the Jupyter notebook proxy url is retrieved for the cluster or not. The default value is yes. If this is present and set to no, then connection information is not collected. The supported values are: yes, no.
-1. ***MINIMIZE_TERRAFORM_LOGGING***. This controls the verbosity of terraform logs. When any value is set for this parameter, the terraform output is redirected to a local file and not printed on syserr. The log file is then uploaded to storage account. Any value can be set for this parameter, e.g.: yes, true.
-1. ***NETWORK_CONFIG***. This controls the VPC type to be used for the MIG. The supported values are default_network, new_network and multi_nic_network. The default value is default_network. The behavior is 
+5. ***SHOW_PROXY_URL***. This controls if the Jupyter notebook proxy url is retrieved for the cluster or not. The default value is yes. If this is present and set to no, then connection information is not collected. The supported values are: yes, no.
+6. ***MINIMIZE_TERRAFORM_LOGGING***. This controls the verbosity of terraform logs. When any value is set for this parameter, the terraform output is redirected to a local file and not printed on syserr. The log file is then uploaded to storage account. Any value can be set for this parameter, e.g.: yes, true.
+7. ***NETWORK_CONFIG***. This controls the VPC type to be used for the MIG. The supported values are default_network, new_network and multi_nic_network. The default value is default_network. The behavior is 
     -  __default_network__: MIG uses the default VPC in the project.
     -  __new_network__: A new VPC is created for the MIG.
     -  __multi_nic_network__: New VPCs are created and used by all the VMs in the MIG. By default 5 new VPCs are created and 5 NICs are used for the MIG but that value is configurable.
-1. ***ENABLE_OPS_AGENT***. Can be one of:
+8. ***ENABLE_OPS_AGENT***. Can be one of:
     - `true` (default): Install Ops Agent with random-backoff retries
     - `false`: Do not install Ops Agent
-1. ***ENABLE_NOTEBOOK***. Can be one of:
+9. ***ENABLE_NOTEBOOK***. Can be one of:
     - `true` (default): Sets up jupyter notebook for the vm instances.
     - `false`: Do not set up jupyter notebook.
```
Thanks for catching this. I'll update it.

sdlin reviewed (approved):
Are these 3 independent changes?

Or are they somehow dependent on one another?  Like is there something about the GKE version that requires some of the other changes?

Approved, pending a revert of the inadvertent markdown number change.

soumyapani commented:
> 

Yes. 
1. first change is setting GKE version to 1.25. We will need to update this version in future so I am considering making it an input as well.
2. Removing PSP and istio. These are blocking GPU driver installation and removed in 1.25. So I removed those blocks.
3. Removed manual IP range allocation since GKE takes care of that automatically.
